pkg/cmd/server/logs.go

```diff
@@ -120,7 +148,57 @@ func readServerLogFile() error {
 	}
 }
 
-func init() {
-	logsCmd.Flags().BoolVarP(&followFlag, "follow", "f", false, "Follow logs")
-	logsCmd.Flags().BoolVar(&fileFlag, "file", false, "Read logs from local server log file")
+func getLogFiles(dir string) ([]string, error) {
+	files, err := os.ReadDir(dir)
+	if err != nil {
+		return nil, err
+	}
+
+	var logFiles []string
+	for _, file := range files {
+		if file.Name() == "daytona.log" || strings.HasPrefix(file.Name(), "daytona-") && (strings.HasSuffix(file.Name(), ".log") || strings.HasSuffix(file.Name(), ".zip")) {
+			logFiles = append(logFiles, filepath.Join(dir, file.Name()))
+		}
+	}
+
+	return logFiles, nil
+}
+
+func selectLogFile(files []string) (string, error) {
+	var options []huh.Option[string]
+	for _, file := range files {
+		options = append(options, huh.Option[string]{Key: filepath.Base(file), Value: filepath.Base(file)})
+	}
+
+	var selected string
+	prompt := huh.NewSelect[string]().
```

Comment:
The prompt doesn't seem to be styled correctly. It should use the common style that the other selects use.